server/fleet/windows_mdm.go

```diff
@@ -45,12 +45,13 @@ type MDMWindowsConfigProfile struct {
 // Returns an error if these conditions are not met.
 func (m *MDMWindowsConfigProfile) ValidateUserProvided() error {
 	if mdm.GetRawProfilePlatform(m.SyncML) != "windows" {
-		return errors.New("Only <Replace> supported as a top level element. Make sure you don’t have other top level elements.")
+		// it doesn't start with <Replace>, assume it isn't valid XML.
+		return errors.New("The file should include valid XML.")
```

Comment:
I changed that because although it _could_ be that the top-level is not `<Replace>`, it could also be that it is just not XML and it is the more general message in this case.